README.md

```diff
@@ -234,6 +234,8 @@ When using Sauce Labs browsers, you have the option to use the Sauce Connect tun
 
 #### Important Notes on Sauce Connect
 
+- We are using Sauce Connect version of 4.5.4.
```

Comment:
Update to "Using Sauce Connect version `4.5.4`."

Reply:
Addressed